app/policies/organizations/faq_policy.rb

```diff
@@ -0,0 +1,10 @@
+class Organizations::FaqPolicy < ApplicationPolicy
+  pre_check :verify_organization!
+  pre_check :verify_active_staff!
+
+  alias_rule :new?, :create?, :index?, to: :manage?
```

Comment:
Same here, should we add update and destroy?

Reply:
I think I have a better understand now of what's going on after reading the docs a few times. ActionPolicy::Base sets `manage?` as default so that it what `update` and `destroy` use. `new?, create?, index?` are already defined in the superclass by default so the alias is used to set them to `manage?` here.

[ActionPolicy Aliases](https://actionpolicy.evilmartians.io/#/aliases)

Reply:
Thanks @jmilljr24 I too had to check the docs out for that. Not obvious at first!